### PR TITLE
Doc: Improve landing pages

### PR DESCRIPTION
**Pull request by akcano**
This normalizes the landing page structure.

## Docs

* [x] I have checked and added or updated relevant documentation.
* [ ] I have checked and added or updated relevant release notes.
* [x] I have included the technical author in the review.

Or:

* [ ] I confirm the PR has no implications for documentation.

### CODE REVIEW

File: docs/explanation/index.rst [hunk 1]
```diff
@@ -3,15 +3,19 @@
...
 
 Workshops and projects
 ----------------------
 
-The core set of |ws_markup|-related concepts
-such as projects and workshops:
+A *workshop* is a container that enables consistent environment builds.
+It is tied to a definition that lists SDKs and is stored as a `.yaml` file
+under a project directory.
+A *project* is the working directory for tasks like coding, running models,
```
Comment (dmitry-lyfar) [inline docs/explanation/index.rst]: I think this gives a too generalised definition of the project directory, it was hard for me to go to `oh, this is where I have my local git repository`. Specifically, not sure what is meant by `managing repositories`.

Comment (akcano) [reply docs/explanation/index.rst]: Done.

File: docs/reference/sdks.rst [hunk 2]
```diff
@@ -405,6 +273,9 @@ a zero code indicates success.
    The hooks aren't mentioned in the :ref:`definition <ref_sdk_definition>`;
    |ws_markup| automatically enumerates them when packing the SDK.
```
Comment (dmitry-lyfar) [inline docs/reference/sdks.rst]:
```suggestion
   |sdk_markup| automatically enumerates them when packing the SDK.
```

File: docs/reference/sdks.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.. _ref_sdks:
```
Comment (dmitry-lyfar) [inline docs/reference/sdks.rst:1]: ws_markup -> sdk_markup

**Review by dmitry-lyfar — COMMENTED** [review]
The new project definition is better. Left some ws_markup -> sdk_markup notes (would appreciate if you can double check this again, I might have missed some).

**Review by dmitry-lyfar — APPROVED** [review]

**Pull request merged**